src/Ads/AccountService.php

```diff
@@ -218,6 +218,7 @@ public function disconnect() {
 		$this->options->delete( OptionsInterface::ADS_ID );
 		$this->options->delete( OptionsInterface::ADS_SETUP_COMPLETED_AT );
 		$this->options->delete( OptionsInterface::CAMPAIGN_CONVERT_STATUS );
+		$this->options->delete( OptionsInterface::CUSTOMER_DATA_TERMS );
```

Comment:
Is there a particular reason why we keep the `ENHANCED_CONVERSION_STATUS`? If they connect with a different account later we don't want to keep a copy of the previous status.